packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -1635,7 +1673,16 @@ flutter:
 
       const FileSystemException e = FileSystemException('', '', OSError('', 999));
 
-      expect(() => handleSymlinkException(e, platform: platform, os: os), returnsNormally);
+      expect(
+        () => handleSymlinkException(
```

Comment:
Can you add 2 more tests that hits this two different instructions? `final String instructions = (version != null && version >= Version(10, 0, 14972)) ...`. Or let me know if I am reading this tests wrong and you are already catching that

Reply:
I didn't really update this behavior or test, I just updated the function signature and made the fields required (the only *real* new test here is `Symlink ERROR_ACCESS_DENIED failures show developers paths that were used`.

Note the test on line 1614 covers newer windows versions.